gibberish_server/models/room.js

```diff
@@ -27,56 +28,77 @@ const ROUND_NUMBER = 10;
 Object.freeze(ROUND_NUMBER);
 
 class Room {
-    constructor(id = uid(), state = STATE.GAME_WAITING, round = 0, players = []) {
+    constructor(id = uid(), state = STATE.GAME_WAITING, round = 0, players = [], startedTime = null, qna = this.generateQuestions()) {
         this.id = id;
         this.state = state;
         this.round = round;
         this.players = players;
-        this.startedTime = null;
+        this.startedTime = startedTime;
+        this.qna = qna
+        this.timer = Math.floor((new Date() - new Date(this.startedTime))/1000)
+    }
+
+    generateQuestions() {
```

Comment:
the generate questions function may get repeated questions. Probably can use a set and keep adding to it until it hits the size 10?

Reply:
ok yea good idea

Reply:
changed in latest commit